src/builtin-addons/core/template-completion-provider.ts

```diff
@@ -62,6 +64,59 @@ const mListHelpers = memoize(listHelpers, { length: 1, maxAge: 60000 }); // 1 se
 
 const mListRoutes = memoize(listRoutes, { length: 1, maxAge: 60000 });
 
+/**
+ * Generates a map of completion label (file name) to array of potential namespaced
+ * paths.
+ * @param addonsMeta addons meta array
+ * @param server Server
+ * @param focusPath currentfocus path
+ * @returns { [key: string]: string[] }
+ */
+function getResultsObj(addonsMeta: Array<AddonMeta>, server: Server, focusPath: ASTPath) {
+  const resultMap: { [key: string]: string[] } = {};
+
+  // Iterate over the addons meta
+  addonsMeta.forEach((addonData: AddonMeta) => {
+    // Get the component registry based on the addon root.
+    // The component registry is a map where the file name is the key and the value are
+    // potential file paths.
+    // Eg: { foo: ['bar/bang/biz/foo.js'] }
+    const addonRegistry = server.getRegistry(addonData.root).component;
+
+    // For each addon meta, generate the namespaced label.
+    Object.keys(addonRegistry).forEach((addonItem) => {
+      const addonFilePaths = addonRegistry[addonItem];
+      const itemLabel = isAngleComponentPath(focusPath) ? normalizeToAngleBracketComponent(addonItem) : addonItem;
+
+      if (!resultMap[itemLabel]) {
+        resultMap[itemLabel] = [];
+      }
+
+      // If file paths are present, then iterate over the filepath and generate the
+      // namespaced label
+      if (addonFilePaths && addonFilePaths.length) {
+        addonFilePaths.forEach((filePath: string) => {
+          // Check if filepath starts with addon's root
+          if (isRootStartingWithFilePath(addonData.root, filePath)) {
+            const rootNameParts = addonData.name.split(path.sep);
```

Comment:
here we always should use `/` instead of `path.sep`, because addon names could not have windows slashes